aws/resource_aws_ebs_volume_test.go

```diff
@@ -89,6 +89,7 @@ func TestAccAWSEBSVolume_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "type", "gp2"),
 					resource.TestCheckResourceAttr(resourceName, "outpost_arn", ""),
 					resource.TestCheckResourceAttr(resourceName, "multi_attach_enabled", "false"),
+					resource.TestCheckResourceAttr(resourceName, "throughput", "0"),
```

Comment:
You shouldnt test this with 0, shouldnt it be 125?

Reply:
This test uses the default `type` ( = `gp2`) and the API returns no value (translates to 0 in state) for `throughput` unless `type` is `gp3`.